backend/src/test/java/de/neuefische/backend/ticket/TicketServiceTest.java

```diff
@@ -20,9 +20,9 @@ class TicketServiceTest {
     @Test
     void getAllTickets() {
 
-        Comment comment = new Comment("1", "comment");
-        Comment comment2 = new Comment("1", "comment");
-        Comment comment3 = new Comment("1", "comment");
+        Comment comment = new Comment("1", "comment","");
+        Comment comment2 = new Comment("1", "comment","");
+        Comment comment3 = new Comment("1", "comment","");
```

Comment:
Extract to a Method to avoid multiple changes for the same thing

```suggestion
       public String createComment() {
            return  new Comment("1", "comment","");
       }
       
       Comment comment1 = createComment();
        Comment comment2 = createComment
        Comment comment3 = createComment
```